content/en/docs/contributing/e2e.md

```diff
@@ -8,9 +8,10 @@ type: "docs"
 cert-manager has an end-to-end test suite that verifies functionality against a
 real Kubernetes cluster.
 
-This document explains how you can run the end-to-end tests yourself.  This is
-useful when you have added or changed functionality in cert-manager and want to
-verify the software still works as expected.
+This test takes around 30 minutes, it will be running on every PR in our cluster.
+It is only adviced to run this locally when you made big changes to the codebase.
```

Comment:
```suggestion
This test takes around 30 minutes, it will be run on every PR in our cluster.
It is only advised to run this locally when you made big changes to the codebase.
```